tests/operators/utils.py

```diff
@@ -105,3 +107,14 @@ def run_dag(dag):
         end_date=DEFAULT_DATE,
         run_at_least_once=True,
     )
+
+
+def get_dag():
+    very_high_number = 99999999
+    return DAG(
+        "test_dag_" + str(random.randint(0, very_high_number)),
+        default_args={
+            "owner": "airflow",
+            "start_date": DEFAULT_DATE,
+        },
```

Comment:
Why are you using this instead of the sample_dag fixture?

Reply:
It's not getting used now, was part of old code. Removing it.